docs/guides/creatingasource.md

```diff
@@ -81,6 +96,9 @@ kubectl create secret generic awscreds \
   --from-literal=secret_access_key=<SECRET_ACCESS_KEY>
 ```
 
+!!! tip "AWS Credentials"
+    Setting up AWS Credentials tips can be found [here](https://docs.triggermesh.io/cloud/sources/awssqs/#api-credentials).
```

Comment:
This phrasing sounds a bit strange to me, and the practice of hiding links behind the "here" word can make them feel obfuscated.

How about this instead?

> Instructions about setting up AWS security credentials can be found in the [documentation page for the Amazon SQS source](https://docs.triggermesh.io/cloud/sources/awssqs/#api-credentials).